.github/workflows/whitesource.yml

```diff
@@ -0,0 +1,33 @@
+name: Whitesource Unified Agent Scan - GO
+
+on:
+  pull_request:
+    branches: [ master* ]
+
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.12.17
+
+    - name: Build
+      run: go build -v ./...
+      
+    - name: WhiteSource Unified Agent Scan
+      env:
+        WS_APIKEY: ${{secrets.WHITESOURCE_ORG_API_KEY}}
+        WS_USERKEY: ${{secrets.WHITESOURCE_PRIORTIZE_USERKEY}}
+        WS_WSS_URL: https://saas.whitesourcesoftware.com/agent
+        WS_PRODUCTNAME: GH_${{github.event.repository.name}}
+        WS_PROJECTNAME: GH_${{github.event.repository.name}}
+      run: |
+        curl -LJO https://unified-agent.s3.amazonaws.com/wss-unified-agent.jar
```

Comment:
Probably want to verify the contents of this download by shasum or something similar, since it will have access to the environment variables. Otherwise, it is possible to have the same thing that happened with the codecov reporter. Not sure if they continuously update that jar or not, but if its possible to pin to a specific version, that might be the best bet.

Reply:
https://honeycomb.jira.com/browse/IS-2662 . Going to create a vendoring in process. Will update the cURL call when I am finished.